dashboard/test/ui/features/star_labs/signin_callout.feature

```diff
@@ -6,6 +6,23 @@ Scenario: Should see callout on 20-hour farmer lesson
   And I wait for the page to fully load
   And element ".login-callout" is visible
 
+@no_mobile
+@no_ie
+# Issue with cookie clearing on IE
```

Comment:
The manual cookie clearing was working in every browser except IE. I can go back and pull this in again to see what the issue was (and then leave a more descriptive comment here).

Reply:
In Saucelabs I get "Failed: Error communicating with the remote browser. It may have died." The cookies work as expected when tested in IE locally, and a rerun always passes. I'm going to keep @no_ie excluded for now.